### DISCUSSION TITLE
Torches

### DISCUSSION THREAD

**Discussion by erox455** [shun126/UE5-DungeonGeneratorDemo#8] [Q&A]
Torches don't spawn for me in 1.5.10. Bug?

**Comment by shun126** [comment]
I used version 1.5.10 on a private game project and was able to generate torches.
I need to find out if the torch is not being generated, or if it is being generated but the coordinates are in an unexpected position.
![Unreal Editor 2024_05_04](https://github.com/shun126/UE5-DungeonGeneratorDemo/assets/23472415/6b57594d-9e89-4716-a98f-5dd81167b9e6)

**Reply by erox455** [reply]
I guess a pillar is a requirement for them to spawn? That's the only way it works. Also...doors spawn next to each other when there is no wall between them, so they lead to the same hallway. That doesn't look right. Could you do something about that? :)

**Reply by erox455** [reply]
And could you please add a customizable spawn chance for sublevels? That would be handy.

**Reply by shun126** [reply]
Yes, A pillar is needed to spawn the torch. I will consider this as we rarely receive comments that the pillars are not needed.
I know there is a problem generating adjacent doors. But I do not have a good solution. I will continue to consider this issue.
I have accepted the establishment of sub-level spawns. I also thought it would be useful.

**Comment by shun126** [comment]
Supported in plugin version 1.5.11.
プラグインバージョン1.5.11で対応しました。

#13
